.github/workflows/build-ton-linux-x86-64-werror.yml

```diff
@@ -42,6 +42,11 @@ jobs:
           -DTON_WERROR_BUILD=On
         ninja -C build
 
+    - name: Check spaces at EOL
+      run: |
+        find -path ./.git -prune -o -path ./third-party -prune -o -path ./LICENSE.LGPL -o -type f ! -name \*.a ! -name \*.so ! -name \*.jar ! -name \*.boc ! -name \*.png -exec sed -i 's/\s\+$//' {} +
```

Comment:
Maybe `sed -i -e 's/[ \t]\+\r$/\r/' -e 's/[ \t]\+$//' $(git grep -Il '' -- ':!^example/android/third_party')`?
This automatically excludes binary files and files in .gitignore and doesn't turn CRLF to LF (which then makes Git to output warning in CI)

Reply:
I made an even better version.